conda-store-server/conda_store_server/server/views/ui.py

```diff
@@ -68,6 +68,30 @@ def ui_list_environments(
     return templates.TemplateResponse("home.html", context)
 
 
+@router_ui.get("/coatl")
+def ui_list_environments(
+    request: Request,
+    search: Optional[str] = None,
+    templates=Depends(dependencies.get_templates),
+    conda_store=Depends(dependencies.get_conda_store),
+    auth=Depends(dependencies.get_auth),
+    server=Depends(dependencies.get_server),
+    entity=Depends(dependencies.get_entity),
+):
+    orm_environments = auth.filter_environments(
+        entity,
+        api.list_environments(conda_store.db, search=search, show_soft_deleted=False),
+    )
+
+    context = {
+        "request": request,
+        "environments": orm_environments.all(),
```

Comment:
Isn't the react api hitting the api endpoints. Why does it need to know the environments?

Reply:
not knowing what's required where for fastapi, i literally just copy pasted. What should get cut? If you want you can also just suggest a more minimal/appropriate version of `ui_coatl` (ie what i should have named the enclosing function).

Reply:
You are just serving static files right? If so you need this function to return a https://fastapi.tiangolo.com/advanced/custom-response/#fileresponse with the path to the html file (assuming it doesn't need templating/auth. You can remove everything from the function since it is not needed. Then make sure that the html file references paths within the static folder. I would suggest using relative links so that you don't have to know the path prefix.